pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -3166,13 +3166,15 @@ public void updateArray(String columnName, Array x) throws SQLException {
       if (sqlType == Types.NUMERIC || sqlType == Types.DECIMAL) {
         return type.cast(getBigDecimal(columnIndex));
       } else {
-        throw new SQLException("conversion to " + type + " from " + sqlType + " not supported");
+        throw new PSQLException(GT.tr("Unsupported type conversion to {1}.", type),
```

Comment:
@marschall , would you please keep `" from " + sqlType ` in the error message?

Reply:
done